Amazon.IonDotnet.Tests/Common/ReaderTestCommon.cs

```diff
@@ -168,11 +168,25 @@ public static void ReadTypeAnnotations_SingleField(IIonReader reader)
             Assert.AreEqual("withannot", reader.CurrentFieldName);
             Assert.AreEqual(18, reader.IntValue());
 
-            string[] annotations = reader.GetTypeAnnotations();
+            var annotations = reader.GetTypeAnnotations();
             Assert.AreEqual(symbols.Count(), annotations.Count());
             Assert.IsTrue(symbols.SequenceEqual(annotations));
         }
 
+        public static void ReadTypeAnnotations_ZeroSymbol(IIonReader reader)
+        {
+            // an int with zero symbol annotation
+            // $0::18
+            reader.MoveNext();
+
+            Assert.AreEqual(IonType.Int, reader.CurrentType);
+            Assert.AreEqual(18, reader.IntValue());
+
+            string[] annotations = reader.GetTypeAnnotations();
```

Comment:
This should throw `UnknownSymbolException`. No symbols with unknown text, including symbol zero, are accessible via APIs that return strings.

Reply:
Updated assert UnknownSymbolException